templates/publishing/src/blog-app.html

```diff
@@ -29,7 +29,6 @@
 <link rel="import" href="../bower_components/app-layout/app-header/app-header.html">
 <link rel="import" href="../bower_components/app-layout/app-header-layout/app-header-layout.html">
 <link rel="import" href="../bower_components/app-layout/app-toolbar/app-toolbar.html">
-<link rel="import" href="../bower_components/app-layout/app-scroll-position/app-scroll-position.html">
```

Comment:
Let's remove this old [`<app-scrollpos-control>`](https://github.com/PolymerElements/app-layout/blob/4dd4304851cc3b32588b5ecc6015697bf6305f73/templates/publishing/src/blog-app.html#L168-L170) too.